interface/rest/models/shop.go

```diff
@@ -0,0 +1,8 @@
+package rest
+
+type Shop struct {
+	ID   string  `json:"id"`
+	Name string  `json:"name"`
+	lat  float32 `json:"lat"`
+	lng  float32 `json:"long"`
+}
```

Comment:
Just brainstorming some other helpful information about a shop:

- Contact information
- Logo?
- Deals/Promotions (potentially for later, not MVP)

It seems address is missing from this model, but in theory, you'd want to return this to the user.

Reply:
Address added, thanks! 👍 

The rest will be coming in the future... But good points!